master/static/migrations/20230208142448_add-update-modified-time-user-table.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE FUNCTION public.set_modified_time ()
+    RETURNS TRIGGER
+    AS $$
+BEGIN
+    IF (TG_OP = 'UPDATE') THEN
+        NEW.modified_at := now();
+        RETURN NEW;
+    END IF;
+END;
+$$
+LANGUAGE plpgsql;
+
+CREATE TRIGGER autoupdate_users_modified_at
+    BEFORE UPDATE ON users
```

Comment:
nit question:  can't remember if there is a `created_at` column, but if there isn't one wdy think about `BEFORE INSERT OR UPDATE ON` in case the insert doesn't put in a datetime?

Reply:
There isn't `created_at` in `users` table.
Currently, `modified_at` db schema looks like this
`modified_at timestamp without time zone DEFAULT CURRENT_TIMESTAMP`
so i don't think we need `INSERT` here. Also, this function could be reused for other `modfied_at`, so i think only `UPDATE` is sufficient.